db/models.py

```diff
@@ -41,8 +46,70 @@ def __str__(self) -> str:
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(to=CinemaHall, on_delete=models.CASCADE)
-    movie = models.ForeignKey(to=Movie, on_delete=models.CASCADE)
+    cinema_hall = models.ForeignKey(to="CinemaHall", on_delete=models.CASCADE)
+    movie = models.ForeignKey(to="Movie", on_delete=models.CASCADE)
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to="User", on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    row = models.PositiveIntegerField()
+    seat = models.PositiveIntegerField()
+    movie_session = models.ForeignKey(to="MovieSession",
+                                      on_delete=models.CASCADE,
+                                      related_name="ticket")
```

Comment:
wrong indentation, you have the right one when creating new tickets:
Ticket.objects.create(
                row=ticket["row"],
                seat=ticket["seat"],
                movie_session=movie,
                order=order
            )
like this